src/main/java/it/gov/pagopa/payhub/activities/dto/treasury/FlussoTesoreriaPIIDTO.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.payhub.activities.dto.treasury;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class FlussoTesoreriaPIIDTO {
```

Comment:
why PII? the segregation should be hidden to activities, it's something related to classification service (moreover, Treasury will not more have PII data)